### PR TITLE
Clean up the JobRequirements

### PR DESCRIPTION
**Pull request by 778b**
## About the PR 

Closes https://github.com/space-wizards/space-station-14/issues/13348

**Changelog**

- Just clean up the JobRequirements

### CODE REVIEW

File: Content.Shared/Roles/JobRequirements.cs [hunk 1]
```diff
@@ -49,6 +59,11 @@ public sealed class OverallPlaytimeRequirement : JobRequirement
         /// How long (in seconds) this requirement is.
         /// </summary>
         [DataField("time")] public TimeSpan Time;
+
+        /// <summary>
+        /// Should the condition be inverted?.
```
Comment (AJCM-git) [inline Content.Shared/Roles/JobRequirements.cs:64]: use     `/// <inheritdoc/>`

File: Content.Shared/Roles/JobRequirements.cs [hunk 2]
```diff
@@ -40,6 +45,11 @@ public sealed class RoleTimeRequirement : JobRequirement
...
```
Comment (AJCM-git) [inline Content.Shared/Roles/JobRequirements.cs:50]: same

File: Content.Shared/Roles/JobRequirements.cs [hunk 3]
```diff
@@ -7,24 +7,29 @@
...
         [DataField("department", customTypeSerializer: typeof(PrototypeIdSerializer<DepartmentPrototype>))]
         public string Department = default!;
 
         /// <summary>
         /// How long (in seconds) this requirement is.
         /// </summary>
         [DataField("time")] public TimeSpan Time;
+
+        /// <summary>
+        /// Should the condition be inverted?.
```
Comment (AJCM-git) [inline Content.Shared/Roles/JobRequirements.cs:30]: Be more concrete than this, declare what is the purpose of the condition being inverted

**Review by AJCM-git — CHANGES_REQUESTED** [review]

File: Content.Shared/Roles/JobRequirements.cs [hunk 4]
```diff
@@ -7,24 +7,35 @@
...
 
         /// <summary>
         /// How long (in seconds) this requirement is.
         /// </summary>
         [DataField("time")] public TimeSpan Time;
+
+        /// <summary>
+        /// If property is <c>True</c> Requirement will inverted
+        /// and it disallow pick this job if condition was true,<br />
+        /// and it allow pick if condition was false.
```
Comment (metalgearsloth) [inline Content.Shared/Roles/JobRequirements.cs:32]:
```suggestion
        /// If true, requirement will return false if playtime above the specified time.
```

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**